src/components/TaskRunListView/TaskRunListRow.tsx

```diff
@@ -1,27 +1,41 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import { PipelineRunLabel } from '../../consts/pipelinerun';
 import { RowFunctionArgs, TableData } from '../../shared/components/table';
 import { Timestamp } from '../../shared/components/timestamp/Timestamp';
 import { TaskRunKind } from '../../types/task-run';
 import { taskRunStatus } from '../../utils/pipeline-utils';
+import { useWorkspaceInfo } from '../../utils/workspace-context-utils';
 import { StatusIconWithText } from '../topology/StatusIcon';
 import { taskRunTableColumnClasses } from './TaskRunListHeader';
 
-const TaskRunListRow: React.FC<RowFunctionArgs<TaskRunKind>> = ({ obj }) => (
-  <>
-    <TableData className={taskRunTableColumnClasses.name}>{obj.metadata.name}</TableData>
-    <TableData className={taskRunTableColumnClasses.task}>
-      {obj.spec.taskRef?.name ?? '-'}
-    </TableData>
-    <TableData className={taskRunTableColumnClasses.started}>
-      <Timestamp timestamp={obj.status?.startTime} />
-    </TableData>
-    <TableData className={taskRunTableColumnClasses.status}>
-      <StatusIconWithText dataTestAttribute="taskrun-status" status={taskRunStatus(obj)} />
-    </TableData>
-    <TableData className={taskRunTableColumnClasses.kebab}>
-      <div />
-    </TableData>
-  </>
-);
+const TaskRunListRow: React.FC<RowFunctionArgs<TaskRunKind>> = ({ obj }) => {
+  const { workspace } = useWorkspaceInfo();
+  const applicationName = obj.metadata?.labels[PipelineRunLabel.APPLICATION];
+  return (
+    <>
+      <TableData className={taskRunTableColumnClasses.name}>
+        <Link
+          to={`/stonesoup/workspaces/${workspace}/applications/${applicationName}/taskruns/${obj.metadata?.name}`}
+          title={obj.metadata?.name}
```

Comment:
`title` isn't necessary as it provides no additional information that the link text doesn't already provide.